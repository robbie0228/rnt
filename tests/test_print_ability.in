abilities
ability 4 B
abilities
move a down
abilities
ability 1 3 4
abilities
move C up
abilities
ability 3 a
abilities
ability 4 C
abilities
ability 5 h
abilities 
move a down
abilities
ability 3 B
ability 4 h
abilities
