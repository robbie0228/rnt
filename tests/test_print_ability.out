Player 1's Abilities:
1: Firewall, Unused
2: Download, Unused
3: Link Boost, Unused
4: Scan, Unused
5: Polarize, Unused
Player 1's Abilities:
1: Firewall, Unused
2: Download, Unused
3: Link Boost, Unused
4: Scan, Used
5: Polarize, Unused
Player 2's Abilities:
1: Firewall, Unused
2: Download, Unused
3: Link Boost, Unused
4: Scan, Unused
5: Polarize, Unused
Player 2's Abilities:
1: Firewall, Used
2: Download, Unused
3: Link Boost, Unused
4: Scan, Unused
5: Polarize, Unused
Player 1's Abilities:
1: Firewall, Unused
2: Download, Unused
3: Link Boost, Unused
4: Scan, Used
5: Polarize, Unused
Player 1's Abilities:
1: Firewall, Unused
2: Download, Unused
3: Link Boost, Used
4: Scan, Used
5: Polarize, Unused
invalid ability
Only use one ability per turn
Player 2's Abilities:
1: Firewall, Used
2: Download, Unused
3: Link Boost, Unused
4: Scan, Unused
5: Polarize, Unused
Only use one ability per turn
